packages/nginx/data_stream/access/elasticsearch/ingest_pipeline/default.yml

```
@@ -21,7 +21,7 @@ processors:
       patterns:
       - (%{NGINX_HOST} )?"?(?:%{NGINX_ADDRESS_LIST:nginx.access.remote_ip_list}|%{NOTSPACE:source.address})
         - (-|%{DATA:user.name}) \[%{HTTPDATE:nginx.access.time}\] "%{DATA:nginx.access.info}"
-        %{NUMBER:http.response.status_code:long} %{NUMBER:http.response.body.bytes:long}
+        %{NUMBER:http.response.status_code:long} %{NUMBER:http.response.body.bytes:long}( %{NUMBER:nginx.access.response_time})?
```

```suggestion
        %{NUMBER:http.response.status_code:long} %{NUMBER:http.response.body.bytes:long} (?:%{NUMBER:nginx.access.response_time})?
```
Suggested changes:
- Add a space between the groups, instead of a leading space inside the group
- For every optional field, we've used a non-capturing  group like starting with`(?:`

These can easily be overlooked but follows the convention and still parses the logs correctly.